core/src/basic.rs

```diff
@@ -81,8 +89,8 @@ pub enum BasicState {
 /// - `state`: Game state
 #[derive(Clone, Debug)]
 pub struct BasicGame {
-    users: Vec<UserId>,
-    state: BasicState,
+    pub users: Vec<UserId>,
```

Comment:
Don't make it public
```suggestion
    users: Vec<UserId>,
```